src/types/common/contactPoint.ts

```
@@ -25,55 +23,58 @@
       use: z.enum(ContactPointUses),
     }),
     // Other (catch-all with basic validation)
     z.object({
       system: z.literal("other"),
-      value: z.string().min(1, { message: t("required") }),
+      value: z.string().min(1, { message: t("field_required") }),
       use: z.enum(ContactPointUses),
     }),
-  ]);
+  ]),
+});
```

ESLint

[error] Insert `⏎`

(prettier/prettier)